app/components/admin/user_booking_stats_component.html.slim

```diff
@@ -1 +1 @@
-strong #{confirmed_bookings_count} confirmed bookings
```

Comment:
This component doesn't follow BEM methodology. Needs more thought re components. But it will do for now.